pipeline/etl.py

```diff
@@ -13,6 +13,22 @@
 EMAIL_REGEX = """(?:[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*|\"(?:[\\x01-\\x08\\x0b\\x0c\\x0e-\\x1f\\x21\\x23-\\x5b\\x5d-\\x7f]|\\\\[\\x01-\\x09\\x0b\\x0c\\x0e-\\x7f])*\")@(?:(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?|\\[(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?|[a-z0-9-]*[a-z0-9]:(?:[\\x01-\\x08\\x0b\\x0c\\x0e-\\x1f\\x21-\\x5a\\x53-\\x7f]|\\\\[\\x01-\\x09\\x0b\\x0c\\x0e-\\x7f])+)\\])"""
 
 
+def lambda_handler(event, context):
+    """Runs the ETL pipeline when the lambda is invoked"""
+    try:
+        main()
+        return {
+            "statuscode": 200,
+            "body": "ETL pipeline executed successfully!"
+        }
+    except Exception as e:
```

Comment:
Could we be able to be more specific with the kind of exception here?

Reply:
If there's many different kinds of exceptions that could occur here, should we pylint disable it?

Reply:
I will pylint disable it, having a general exception would be better as there are errors we may not be able to catch with a specfic exception.